dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -91,6 +91,7 @@ namespace dwa_local_planner {
       limits.prune_plan = config.prune_plan;
       limits.trans_stopped_vel = config.trans_stopped_vel;
       limits.rot_stopped_vel = config.rot_stopped_vel;
+      limits.stopping_scaling_percent = config.stopping_scaling_percent;
```

Comment:
if this config is going to make it to the GRM, need to add it to an env variable etc

Reply:
That is in the ros part of this review